megatron/core/models/common/embeddings/rotary_pos_embedding.py

```diff
@@ -12,6 +12,7 @@
 
 import torch
 from torch import Tensor, nn
+import transformer_engine.pytorch.cpp_extensions as tex
```

Comment:
I think you might want to do similar things to how they import apex as it is possible that TE might not be there in some dockers. Try to import the rotary embedding from TE and if it is available then `HAVE_APPLY_ROPE_FUSION` could be True on ROCm. You can refer to this example: https://github.com/NVIDIA/Megatron-LM/blob/73e7b58e79df9da521ff31d74053579b7a060c7e/megatron/core/transformer/transformer_block.py#L22-L44